components/autogen/src/format-pages.txt

```diff
@@ -437,6 +438,13 @@ presenceRating = Fair
 utilityRating = Good
 reader = MicromanagerReader
 mif = true
+notes = - Bio-Formats will recognize a :file:`*metadata.txt` file as part of a \n
+Micro-Manager fileset if pointed at it and will load the fileset including \n
```

Comment:
These lines need to be indented (2 spaces) else the generated rst fails to build with

```
     [exec] Warning, treated as error:
     [exec] /opt/ome/bioformats/docs/sphinx/formats/micro-manager.txt:54: WARNING: Bullet list ends without a blank line; unexpected unindent.
```